pkg/operation/botanist/component/etcd/etcd.go

```diff
@@ -599,6 +605,9 @@ func (e *etcd) Get(ctx context.Context) (*druidv1alpha1.Etcd, error) {
 func (e *etcd) SetSecrets(secrets Secrets)                 { e.secrets = secrets }
 func (e *etcd) SetBackupConfig(backupConfig *BackupConfig) { e.backupConfig = backupConfig }
 func (e *etcd) SetHVPAConfig(hvpaConfig *HVPAConfig)       { e.hvpaConfig = hvpaConfig }
+func (e *etcd) SetETCDConfig(config *configv1alpha1.ETCDConfig) {
```

Comment:
Why do we need such setter unction? I think the config is static, so you can pass the etcd config while calling `New`

Reply:
Yes. We don't need any changes in this etcd at all. Those are for deployment of etcd components, not druid. I needed config only for Druid, which I already took care. My bad. I rectified it.